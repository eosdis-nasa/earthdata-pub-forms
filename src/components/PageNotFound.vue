<template lang="html">
  <div role="main">
    Error - Page Not Found
  </div>
</template>
<script>
export default {
  name: 'PageNotFound'
}
</script>
<style scoped>
</style>