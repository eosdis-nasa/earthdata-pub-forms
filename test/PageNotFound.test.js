import { mount} from "@vue/test-utils";
import PageNotFound from "@/components/PageNotFound"

describe("PageNotFound", () => {
    it('renders correctly', () => {
        const wrapper = mount(PageNotFound)
        expect(wrapper.element).toMatchSnapshot()
      })
})