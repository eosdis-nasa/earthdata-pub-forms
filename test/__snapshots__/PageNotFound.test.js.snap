// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders correctly 1`] = `
<div
  role="main"
>
  <span>
    
    Error - Page Not Found
  
  </span>
</div>
`;
